.buildkite/scripts/lifecycle/ci_stats_start.js

```diff
@@ -0,0 +1,30 @@
+const { execSync } = require('child_process');
+const ciStats = require('./ci_stats');
+
+(async () => {
+  try {
+    const build = await ciStats.post('/v1/build', {
+      jenkinsJobName: process.env.BUILDKITE_PIPELINE_NAME,
```

Comment:
I'm assuming this is intentional, is the plan to eventually migrate at API v2?

Reply:
Intentional, yes. @spalger and I haven't explicitly talked about it in a while, but yeah, we eventually want an API that doesn't have Jenkins in the property names